ember-flight-icons/addon/components/flight-icon.js

```diff
@@ -44,14 +44,16 @@ export default class FlightIconComponent extends Component {
   }
 
   /**
-   * Determines the icon's `display` property
-   *
-   * @param display
-   * @returns the value of `isInlineBlock` if set
-   * @default true
+   * Get a class to apply to the icon based on the display argument.
+   * @method FlightIcon#displayClass
+   * @return {string} The css class to apply to the SVG.
    */
-  get display() {
-    return this.args.isInlineBlock ?? true;
+  get displayClass() {
```

Comment:
The previous logic above defaulted to `true` which meant that by default `flight-icon-display-inline` was added, but with this change that is no longer the case which seems like it would be a breaking change. Is that intentional?

Reply:
yes, nice catch! I have fixed the qunit tests (some were broken, looking for a class that was not there) and added a new one to cover this case. check my new implementation of the `displayClass` method, see if it's OK (but I am open to more elegant solutions, if you have one)

Reply:
This seems ok to me. I'm a little fuzzy on why we don't want to allow inline-block in liquid mode but i'm super rusty here so will defer to your expertise.

Reply:
with `display: block`:
<img width="282" alt="screenshot_309" src="https://user-images.githubusercontent.com/686239/140408261-3cd3205a-ddc2-42a4-998d-45ca47e208f9.png">

with `display: inline-block`:
<img width="269" alt="screenshot_310" src="https://user-images.githubusercontent.com/686239/140408302-caa712e4-959f-4419-9cae-50b46ed8e56d.png">

(the pink box is the container)